mkt/webapps/models.py

```diff
@@ -2589,56 +2589,15 @@ class AppFeatures(ModelBase, DynamicBoolFieldsMixin):
     class Meta:
         db_table = 'addons_features'
 
-    def __unicode__(self):
-        return u'Version: %s: %s' % (self.version.id, self.to_signature())
-
-    def set_flags(self, signature):
-        """
-        Sets flags given the signature.
-
-        This takes the reverse steps in `to_signature` to set the various flags
-        given a signature. Boolean math is used since "0.23.1" is a valid
-        signature but does not produce a string of required length when doing
-        string indexing.
-        """
-        fields = self._fields()
-        # Grab the profile part of the signature and convert to binary string.
-        try:
-            profile = bin(int(signature.split('.')[0], 16)).lstrip('0b')
-            n = len(fields) - 1
-            for i, f in enumerate(fields):
-                setattr(self, f, bool(int(profile, 2) & 2 ** (n - i)))
-        except ValueError as e:
-            log.error(u'ValueError converting %s. %s' % (signature, e))
-
-    def to_signature(self):
```

Comment:
Yeah, it confused me as well ! That's why I removed it, trying to keep things simpler.